src/cinder/Log.cpp

```diff
@@ -318,6 +409,44 @@ void LoggerNSLog::write( const Metadata &meta, const string &text )
 	NSLog( @"%@ %@", metaDataNs, textNs );
 }
 
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerNSLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerSysLog::LoggerSysLog()
+{
+	// Most general approach for determining app name.  Perhaps there is something more cinder specific?
+	// https://developer.apple.com/library/mac/qa/qa1544/_index.html
+
+	NSString *bundlePath = [[NSBundle mainBundle] bundlePath];
+	NSString *appName = [[NSFileManager defaultManager] displayNameAtPath: bundlePath];
+
+	const char *cAppName = [appName UTF8String];
+	openlog( cAppName, ( LOG_CONS | LOG_PID), LOG_USER );
+}
+
+LoggerSysLog::~LoggerSysLog()
+{
+	closelog();
+}
+
+void LoggerSysLog::write( const Metadata &meta, const string &text )
+{
+	// TODO: code duplication
+	int sysLevel;
+	switch ( meta.mLevel ) {
+		case LEVEL_VERBOSE:		sysLevel = LOG_DEBUG;	break;
+		case LEVEL_INFO:		sysLevel = LOG_INFO;	break;
+		case LEVEL_WARNING:		sysLevel = LOG_WARNING;	break;
+		case LEVEL_ERROR:		sysLevel = LOG_ERR;		break;
+
+		// debatable if FATAL is a LOG_CRIT or LOG_ALERT
+		case LEVEL_FATAL:		sysLevel = LOG_CRIT;	break;
+		default: CI_ASSERT_NOT_REACHABLE();
+	}
+	syslog(sysLevel, "%s %s", meta.toString().c_str(), text.c_str());
```

Comment:
True, but we'll still be filtering log messages via cinder's ci::log::Level mechanism, so verbose things should still get filtered out, in release mode at least. We can think about adding per-logger levels in the future, but that would have to be runtime filtered of course.